src/battle_dome.c

```diff
@@ -0,0 +1,6480 @@
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6043, .y = 0x07, .x = 0x0c},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d47c[] =
+{
+    {.src = 0x6021, .y = 0x04, .x = 0x09},
+    {.src = 0x6023, .y = 0x04, .x = 0x0a},
+    {.src = 0x6047, .y = 0x05, .x = 0x0a},
+    {.src = 0x6023, .y = 0x05, .x = 0x0b},
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6023, .y = 0x07, .x = 0x0c},
+    {.src = 0x6027, .y = 0x08, .x = 0x0c},
+    {.src = 0x6027, .y = 0x09, .x = 0x0c},
+    {.src = 0x6027, .y = 0x0a, .x = 0x0c},
+    {.src = 0x603b, .y = 0x0b, .x = 0x0c},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d4a8[] =
+{
+    {.src = 0x6021, .y = 0x04, .x = 0x09},
+    {.src = 0x6023, .y = 0x04, .x = 0x0a},
+    {.src = 0x6047, .y = 0x05, .x = 0x0a},
+    {.src = 0x6023, .y = 0x05, .x = 0x0b},
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6023, .y = 0x07, .x = 0x0c},
+    {.src = 0x6027, .y = 0x08, .x = 0x0c},
+    {.src = 0x6027, .y = 0x09, .x = 0x0c},
+    {.src = 0x6027, .y = 0x0a, .x = 0x0c},
+    {.src = 0x603b, .y = 0x0b, .x = 0x0c},
+    {.src = 0x602c, .y = 0x0b, .x = 0x0d},
+    {.src = 0x602d, .y = 0x0b, .x = 0x0e},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d4dc[] =
+{
+    {.src = 0x6021, .y = 0x06, .x = 0x09},
+    {.src = 0x6021, .y = 0x06, .x = 0x0a},
+    {.src = 0x6027, .y = 0x05, .x = 0x0a},
+    {.src = 0x6043, .y = 0x05, .x = 0x0b},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d4ec[] =
+{
+    {.src = 0x6021, .y = 0x06, .x = 0x09},
+    {.src = 0x6021, .y = 0x06, .x = 0x0a},
+    {.src = 0x6027, .y = 0x05, .x = 0x0a},
+    {.src = 0x6023, .y = 0x05, .x = 0x0b},
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6043, .y = 0x07, .x = 0x0c},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d508[] =
+{
+    {.src = 0x6021, .y = 0x06, .x = 0x09},
+    {.src = 0x6021, .y = 0x06, .x = 0x0a},
+    {.src = 0x6027, .y = 0x05, .x = 0x0a},
+    {.src = 0x6023, .y = 0x05, .x = 0x0b},
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6023, .y = 0x07, .x = 0x0c},
+    {.src = 0x6027, .y = 0x08, .x = 0x0c},
+    {.src = 0x6027, .y = 0x09, .x = 0x0c},
+    {.src = 0x6027, .y = 0x0a, .x = 0x0c},
+    {.src = 0x603b, .y = 0x0b, .x = 0x0c},
+};
+
+static const struct UnkStruct_860DD10 gUnknown_0860d534[] =
+{
+    {.src = 0x6021, .y = 0x06, .x = 0x09},
+    {.src = 0x6021, .y = 0x06, .x = 0x0a},
+    {.src = 0x6027, .y = 0x05, .x = 0x0a},
+    {.src = 0x6023, .y = 0x05, .x = 0x0b},
+    {.src = 0x6027, .y = 0x06, .x = 0x0b},
+    {.src = 0x6047, .y = 0x07, .x = 0x0b},
+    {.src = 0x6023, .y = 0x07, .x = 0x0c},
+    {.src = 0x6027, .y = 0x08, .x = 0x0c},
+    {.src = 0x6027, .y = 0x09, .x = 0x0c},
+    {.s
```

Comment:
This isn't a bug. It's calculating the total points for all 3 mons on each side. Read the code more carefully.

Reply:
Sadly, it's a bug. It should be cleared once the match's result is determined.